.github/workflows/release.yml

```diff
@@ -84,30 +84,33 @@ jobs:
           commit: ${{ github.sha }}
           path: ${{github.workspace}}
 
-      - name: "${{ matrix.component.Name }} -- Publish release image to GHCR"
+      - name: "${{ matrix.component.name }} -- Publish release image to GHCR"
         working-directory: ${{github.workspace}}
         env:
-          VAPP_IMAGE: ${{ matrix.component.Name }}-multiarch/${{ matrix.component.Name }}.tar
-          VAPP_NAME: ${{ matrix.component.Name }}
+          VAPP_IMAGE: ${{ matrix.component.name }}-multiarch/${{ matrix.component.name }}.tar
+          VAPP_NAME: ${{ matrix.component.name }}
           VAPP_VERSION: ${{ steps.get_version.outputs.version-without-v }}
           REGISTRY: "ghcr.io/${{steps.github-repository-name-case-adjusted.outputs.lowercase}}"
         run: |
           tag=$(echo docker://$REGISTRY/$VAPP_NAME:$VAPP_VERSION | tr '[:upper:]' '[:lower:]')
           echo "Copy vApp image $VAPP_IMAGE to '$tag'"
           skopeo copy --all  oci-archive:$VAPP_IMAGE "$tag"
 
-      - name: ${{ matrix.component.Name }} -- Download CI artifacts
+      - name: ${{ matrix.component.name }} -- Download CI artifacts
         uses: dawidd6/action-download-artifact@v2
         with:
           workflow: ci.yml
           workflow_conclusion: success
           path: ./ci-artifacts
 
-      - name: ${{ matrix.component.Name }} -- Upload assets
+      - name: ${{ matrix.component.name }} -- Upload assets
         uses: softprops/action-gh-release@v1
         if: startsWith(github.ref, 'refs/tags/')
         with:
-          files: ./ci-artifacts/AppManifest/AppManifest.json
+          files: |
+            ./ci-artifacts/AppManifest/AppManifest.json
+            ./ci-artifacts/Podspec/Podspec.yaml
+            ./ci-artifacts/HelmChart
```

Comment:
these files are not generated during the CI workflow so this won't work

Reply:
Every workflow is common in Python and C++ template - except CI workflows.
Is generation triggered by this step in Python CI:
```
      - name: Configure K3D cluster and install runtime containers in K3D cluster
        run: velocitas exec runtime-k3d up
```
?

Reply:
this works because its the same like in python. the files are generated in the build-docker-image.yml

Reply:
no it does not work, see https://github.com/eclipse-velocitas/vehicle-app-cpp-template/actions/runs/5080982468

there are no artifacts which can be downloaded. These artifacts are build in build-docker-image.yml, which is called in CI of python but currently not in cpp

Reply:
ooookay you are right - than that should work? But: Why does nobody complain about the AppManifest artifact before? it actually also was a requirement and it does. not seem to work in cpp

Reply:
good question, most probably because CPP is not that well recognized...

The question is rather, why do we not call the build-docker-image workflow here? Is there a reason at all?

Reply:
most probably there was simply no reason to build the container always, as there are also no integration tests yet. we need to build the app for the unit tests and there we can't use the container...

so now we need to take a decision, either make the ci workflow much longer, since we need to build the app twice or move the generations directly into the ci workflow...

Reply:
or we just download the files from the multi-arch build..

Reply:
im fine with anything which is not extending pipeline time :P

Reply:
Solution is to take it from multiarch build.